src/components/scene/pool.js

```diff
@@ -0,0 +1,103 @@
+var debug = require('../../utils/debug');
+var register = require('../../core/component').registerComponent;
+
+var warn = debug('components:pool:warn');
+
+/**
+ * Pool component.
+ * It initialized a pool of entities that will be reused.
+ * It avoids creating and destroying the same kind entities
+ * in dynamic scenes. Useful for example in a game where you want
+ * to reuse enemies entities
+ *
+ * @member {array} poolEls - Available entities in the pool.
+ * @member {array} pooledEls - Entities of the pool in use.
+ *
+ */
+module.exports.Component = register('pool', {
```

Comment:
When an application requires pools I expect to often see multiple instances `pool__bullets`, `pool__enemies`... If we rename `pool` to `entity-pool` you will end up with `entity-pool__enemies` , `entity-pool_bullets`, Is not that to verbose? @fernandojsg